src/graph_compute/bsp_scheduling.h

```diff
@@ -14,14 +14,18 @@ static void init_scheduling(vertex_t * const nodes, const vid_t cntNodes,
   // no-op
 }
 
-static void execute_round(const int round, vertex_t * const nodes, const vid_t cntNodes,
+static void execute_round(const int numRounds, vertex_t * const nodes,
```

Comment:
If we're changing the meaning of the first parameter, we should also rename the method -- `execute_round` doesn't sound like it executes all rounds.